src/components/accordion/accordion.pw.tsx

```diff
@@ -121,34 +121,37 @@ test.describe("should render Accordion component", () => {
     await expect(accordionContent(page)).toBeVisible();
   });
 
-  (["chevron_down", "dropdown"] as const).forEach((iconType) => {
-    test(`should set iconType to ${iconType} when Accordion row is closed`, async ({
-      page,
-      mount,
-    }) => {
-      await mount(<AccordionComponent iconType={iconType} />);
-
-      await expect(accordionIcon(page)).toHaveAttribute("type", iconType);
-      await expect(accordionIcon(page)).toBeVisible();
-      const transformValue = await getStyle(accordionIcon(page), "transform");
-      await expect(getRotationAngle(transformValue)).toBe(90);
-    });
-  });
-
-  (["chevron_down", "dropdown"] as const).forEach((iconType) => {
-    test(`should set iconType to ${iconType} when Accordion row is open`, async ({
-      page,
-      mount,
-    }) => {
-      await mount(<AccordionComponent iconType={iconType} />);
-
-      await accordionTitleContainer(page).click();
-
-      await expect(accordionIcon(page)).toHaveAttribute("type", iconType);
-      await expect(accordionIcon(page)).toBeVisible();
-      await expect(accordionIcon(page)).toHaveCSS("transform", "none");
-    });
-  });
+  (["chevron_down", "dropdown", "chevron_down_thick"] as const).forEach(
+    (iconType) => {
+      test(`should set iconType to ${iconType} and transform when Accordion row is closed`, async ({
+        page,
+        mount,
+      }) => {
+        await mount(<AccordionComponent iconType={iconType} />);
+
+        await expect(accordionIcon(page)).toHaveAttribute("type", iconType);
+        await expect(accordionIcon(page)).toBeVisible();
+        const transformValue = await getStyle(accordionIcon(page), "transform");
```

Comment:
suggestion(non-blocking): I don't _think_ you need the `await` here. 

```suggestion
        const transformValue = getStyle(accordionIcon(page), "transform");
```

Reply:
I'm afraid it is in fact needed